src/main/java/com/artipie/MeasuredStorage.java

```diff
@@ -169,14 +170,8 @@ public void onComplete() {
     }
 
     @Override
-    public CompletableFuture<Long> size(final Key key) {
-        final long start = System.nanoTime();
-        return this.origin.size(key).thenApply(
-            res -> {
-                this.log("size(%s): %s", key.string(), millisMessage(start));
-                return res;
-            }
-        );
+    public CompletableFuture<? extends Meta> metadata(final Key key) {
+        return this.origin.metadata(key);
```

Comment:
Every method of `MeasuredStorage` logs execution time, I'd suggest logging it here too